FlexMeasures/flexmeasures
# Dynamic account colors

## Description

Summary of the changes introduced in this PR. Try to use bullet points as much as possible.
- Modification to Account Model to hold primary and secondary colors
- Modification to ``` flexmeasures add account ``` command to accept these colors if needed
- Modification to flexmeasures.css ( root is now dynamically being rendered)
- Small fix in flexmeasures.css, there was a comment made, and the comment was not written correctly(probably due to a typo)

## Look & Feel
Here is an account created with the new additions
```flexmeasures add account --name JGreenConsult --primary-color 009933 --secondary-color 0099ff```

![1024-511](https://github.com/user-attachments/assets/61efb687-3c39-40f1-9ccc-cf3ac76f7070)

## How to test

Run the below command using the new additions to the ``` flexmeasures add account ``` command

**NOTE:** The hex color to be parsed should be parsed without a ```#```.

**_For example:_**
✅: --primary-color ff0000
❌: --primary-color #ff0000

**_Command Blueprint:_**
```flexmeasures add account --name <company-name> --primary-color <hex-color> --secondary-color <hex-color>```

**_Full example:_**
```flexmeasures add account --name MrcEnergy --primary-color 0000ff --secondary-color 00ff00 ```


## Further Improvements

Closes #471 

- [✅] I agree to contribute to the project under Apache 2 License. 
- [✅] To the best of my knowledge, the proposed patch is not based on code under GPL or other license that is incompatible with FlexMeasures
